### PR TITLE
{lib}[GCCcore/12.3.0] Guile v3.0.9, gc v8.2.4, libunistring v1.1

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'gc'
+version = '8.2.2'
```
Comment (branfosj) [inline easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb:4]: Is there a reason to not use [8.2.4](https://github.com/ivmai/bdwgc/releases/tag/v8.2.4)?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb:4]: my bad, I was looking at https://hboehm.info/gc/gc_source which has 8.2.2 as the latest one.

Comment (maxim-masterov) [reply easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb]: updated versions of `gc` and `libatomic`

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/02ba053b07d7d2c016cf68f341acf3e9 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**